integration/ui/satellite/e2e_onboarding_test.go

```diff
@@ -0,0 +1,222 @@
+// Copyright (C) 2021 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellite
+
+import (
+	"testing"
+	"time"
+
+	"github.com/go-rod/rod"
+	assert2 "github.com/stretchr/testify/assert"
+	"github.com/zeebo/assert"
+
+	"storj.io/common/testcontext"
+	"storj.io/storj/integration/ui/uitest"
+	"storj.io/storj/private/testplanet"
+)
+
+func TestE2eUserCreateLoginAccessInBrowser(t *testing.T) {
+	uitest.Run(t, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet, browser *rod.Browser) {
+		loginPageURL := planet.Satellites[0].ConsoleURL() + "/login"
+		email := "test@dtsff4xxxrvfdwqd0.com"
+		nickname := "test2"
+		pass := "123qwe"
+		accessGrantName := "qwerty"
+		page := browser.MustPage(loginPageURL)
+		page.MustSetViewport(1350, 600, 1, false)
+
+		// new user creation
+		page.MustElement("a.login-area__content-area__forgot-container__link.register-link").MustClick()
+		time.Sleep(1 * time.Second)
```

Comment:
Please remove Sleep and wait for some element on that page e.g. 
`confirmPage := page.MustElement(".register-area__intro-area__title").MustText()`
`require.Contains(t, confirmPage, "Welcome to the decentralized cloud")`